centreon/packages/ui/src/utils/docsURL.ts

```diff
@@ -0,0 +1,31 @@
+interface OnPremDocsURL {
+  majorVersion: string;
+  minorVersion: string;
+}
+
+interface DocsURL {
+  isCloudPlatform: boolean;
+  majorVersion: string;
+  minorVersion: string;
+}
+
+export const getOnPremDocsURL = ({
+  majorVersion,
+  minorVersion
+}: OnPremDocsURL): string => {
+  return `https://docs.centreon.com/docs/${majorVersion}.${minorVersion}/alerts-notifications/resources-status/#search-bar`;
```

Comment:
does it work with 23.10 as well ?

Reply:
Considering that the docs for 23.10 have not been released yet. it will display a 404 page from the docs website. However, this should not be a problem since the Centreon 23.10 has not been released yet either. Right?

Reply:
This should not be a problem yet for the 23.10. If there is a staging documentation for the 23.10, we can redirect the user to the staging documentation.